examples/somersaultlazy.py

```diff
@@ -550,6 +531,8 @@ async def main(args):
 
 args = parser.parse_args()  # deals with the help message handling
 
+#logging.basicConfig(level=logging.INFO)
```

Comment:
Commented out code

Reply:
that's on purpose: if somebody wants to work on this code, this demonstrates how to change the verbosity. Maybe I should extend this a bit...

Reply:
done: 4099dc8709b6c78ed458242300bcca4b9a5fadb8